src/lib/externaldata.py

```diff
@@ -88,18 +90,21 @@ def to_json(self):
 
         return json.dumps(json_data, indent=4)
 
+
 class Checker:
 
     def check(self, external_data):
         raise NotImplementedError()
 
+
 class CheckerRegistry:
 
     _checkers = []
 
     @staticmethod
     def load(checkers_folder):
-        for _unused, modname, _unused in pkgutil.walk_packages([checkers_folder]):
+        for _unused, modname, _unused in pkgutil.walk_packages(
+                [checkers_folder]):
```

Comment:
I don't like opening parenthesis without args in that line. I'd prefer to break after the `in`.